WRO-11453: Custom skin app: Add explanatory comments in the code. (#182)
enactjs/samples: sandstone/custom-skin/README-users.md

## Custom Skin

A sample Enact application that uses custom-skin feature to style the components.

Run `npm install` then `npm run serve` to have the app running on [http://localhost:8080](http://localhost:8080), where you can view it in your browser.

> This app will let you generate a stylesheet that customizes a sandstone application. On the right are all the interactive components, while on the left 

Reviewer: ```suggestion
This app will let you generate a stylesheet that customizes a Sandstone application. On the left you can see all the interactive components, while on the right 
```

is the `Live Preview` area. Any change you make inside the app should be reflected inside `Live Preview`.

> The dropdown on the top of the page allows you to choose between sandstone default skin and a few presets. There is also an option to import your own.

Reviewer: ```suggestion
The dropdown on the top of the page allows you to choose between Sandstone default skin and a few presets. There is also an option to import your own.
```

> The switch near it turns on the auto generation of colors. This feature will generate a skin based on the first two colors listed(the background and text color).

Reviewer: ```suggestion
The switch near it turns on the auto generation of colors. This feature will generate a skin configuration based on the first two colors listed(the background and text color).
```

Reviewer: please choose any other appropriate term that can be used instad of configuration, if you can find a better one
